### ISSUE TITLE
Reboot / re-recruit for labs roles

### ISSUE THREAD

**Issue by rgrp** [okfn/okfn.github.com#367]
These are all volunteer roles supported by Labs community manager at Open Knowledge International

* [ ] define roles
* [ ] prioritise
* [ ] recruit

## General Points

* 6m or 1y term

### Leads and Management Team

* Leads: You are the "chair" of Labs, responsible for team meetings and getting its job done
* Team: ... 

Activities

  * Manage and direct resources
  * Run hangouts
  * Inspire and coordinate labs activity
  * Respond to enquiries

### Advisory Group

Emeritus labs leads and other key figures ...

* clearly defined role / activities (maybe we even split these up and recruit for each one)

### Outreach and Communications

* Blog

**Comment by davidmiller** [comment]
@rgrp Great starting point!
+10 to this initiative :) 

Short-med term wouldn't have bandwidth for anything other than Advisory Group I suspect.

**Comment by jgkim** [comment]
@rgrp +1 (maybe 10 ;)) to this initiative.

**Comment by rgrp** [comment]
@jgkim great stuff - would you be interested in participating in some capacity :-) ?

**Comment by jgkim** [comment]
@rgrp not 100% sure, but maybe a member of the management team or the advisory group? of course, if I'm qualified. ;)

**Comment by danfowler** [comment]
@jgkim @davidmiller still interested?

**Comment by jgkim** [comment]
@danfowler I don't have much time at the moment, but my answer is still "yes."

**Comment by danfowler** [comment]
As a follow-up to our hangout last week ([pad here](https://pad.okfn.org/p/labs-hangouts)), it looks like we have a team forming:

* @davbre 
* @davidmiller (advisory group)
* @jgkim (advisory group)
* @loleg (I assume you are still keen to remain as a lead)
* @mattfullerton (you too :)
* @pdehaye (core datasets managing curator)
* (me) 

Do I have this right?  @flowsta are you interested in contributing, as well?  @andylolz, do you have your  [powers](https://github.com/okfn/datapipes/issues/119#issuecomment-160299957) back ?

As a first task, it would be great to have someone put the [next newsletter](https://github.com/okfn/okfn.github.com/issues/374) together sometime before the beginning of next week using the list of [current submissions](https://github.com/okfn/okfn.github.com/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+newsletter) and any other [interesting activity](https://github.com/okfn/bibserver/pull/261#issuecomment-160114627) around Labs projects.  Any takers?

We could also start planning the next hangout.  It would be great to invite @jbenet as a guest speaker to talk about http://ipfs.io/.

Also, please take a look at this post and leave any suggestions for improvement: https://discuss.okfn.org/t/open-knowledge-labs-outcomes-and-outputs-nov-2015/1594/3

Cheers :sunglasses:

**Comment by andylolz** [comment]
> do you have your [powers](https://github.com/okfn/datapipes/issues/119#issuecomment-160299957) back ?

I do thanks! Will start hacking in earnest on [data pipes](http://datapipes.okfnlabs.org/) in the next week or so.

**Comment by flowsta** [comment]
@danfowler  Yes, I am interested in contributing! I'll take a look to your suggestions

**Comment by jbenet** [comment]
@danfowler thanks! happy to talk about ipfs and whatever is relevant/useful. :+1:

**Comment by todrobbins** [comment]
@rgrp @danfowler what other roles are in need of filling? I'm ready to assist.

**Comment by danfowler** [comment]
Great! Let's chat!  https://gitter.im/okfn/chat

There are a few things that need doing, the most important of which is shutting down our last server and assisting with a migration of all things that were once hosted on the server to Heroku.

https://github.com/okfn/okfn.github.com/issues/457

**Comment by todrobbins** [comment]
Do we need to recruit roles again at this point? @rufuspollock @serahrono @Stephen-Gates 

Or maybe I've lost track of current roles and this is merely a personal issue 😆

**Comment by rufuspollock** [comment]
@todrobbins we do need to re-recruit, that's correct. /cc @serahrono
